modules/default/weather/forecast.njk

```diff
@@ -19,10 +19,16 @@
                 <td class="align-right min-temp">
                     {{ f.minTemperature | roundValue | unit("temperature") }}
                 </td>
-                {% if config.showPrecipitationAmount %}
-                    <td class="align-right bright precipitation">
-                        {{ f.precipitation | unit("precip") }}
-                    </td>
+                {% if (config.showPrecipitationAmount) and (f.precipitation > 0) %}
```

Comment:
Why add that extra case? Cant the precipation be 0 ?

Reply:
You are right and this is ugly. Basically I wanted to ensure precipitation would only be rendered is there actually is a value (and selfishly this was more for my own envcanada code). I will fix this in my envcanada code and this case will be removed.